src/modules/commander/Commander.cpp

```diff
@@ -2108,7 +2108,7 @@ Commander::run()
 
 		} else {
 			if (!status_flags.rc_input_blocked && !status.rc_signal_lost) {
-				mavlink_log_critical(&mavlink_log_pub, "MANUAL CONTROL LOST (at t=%llums)", hrt_absolute_time() / 1000);
+				mavlink_log_critical(&mavlink_log_pub, "MANUAL CONTROL LOST (at t=%" PRIu64 "ms)", hrt_absolute_time() / 1000);
```

Comment:
This might break for the Snapdragon processor. @dagar Have we QuRT building reliably in CI?

Reply:
Disregard, this fixes it the right way around.